ui/lib/ldap/addon/templates/libraries/library/details/configuration.hbs

```diff
@@ -0,0 +1 @@
+<Page::Library::Details::Configuration @model={{this.model}} />
```

Comment:
We've (inconsistently) started labeling component args for model's more explicitly like `@library` instead of `@model`. While it feels more necessary when the model hook has more data on it than a single ember data model, I've still found it helpful when I'm in a component file and trying to track down the model name. 

So no need to change, just offering for consideration!

Reply:
Ya that's a good point. In the case where the route model has multiple things to pass down to the page component I have been naming the args something more descriptive. I guess I figured it was implied that the model would be a library in this case but better not to assume 😁. Since I have been doing it this way for ldap I will keep it consistent but consider more explicit naming in the future.